doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -350,6 +350,17 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-6-0
+
+KRKNWK-18769: Bridge application does not print hyperlink for displaying the setup QR code in logs.
+  This happens, because the log module that displays this log was disabled.
```

Comment:
```suggestion
KRKNWK-18769: :ref:`matter_bridge_app` application does not print the hyperlink for displaying the setup QR code in the log.
  This happens because the log module that displays this log entry has been disabled.
```